[2025-07-06T22:00:23.703+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-06T22:00:23.716+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: practice_etl_pipeline.load_log_data manual__2025-07-06T21:59:26.788588+00:00 [queued]>
[2025-07-06T22:00:23.720+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: practice_etl_pipeline.load_log_data manual__2025-07-06T21:59:26.788588+00:00 [queued]>
[2025-07-06T22:00:23.721+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-06T22:00:23.729+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_log_data> on 2025-07-06 21:59:26.788588+00:00
[2025-07-06T22:00:23.733+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-06T22:00:23.733+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5312) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-06T22:00:23.734+0000] {standard_task_runner.py:63} INFO - Started process 5445 to run task
[2025-07-06T22:00:23.736+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'practice_etl_pipeline', 'load_log_data', 'manual__2025-07-06T21:59:26.788588+00:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmps0jr0t_a']
[2025-07-06T22:00:23.737+0000] {standard_task_runner.py:91} INFO - Job 320: Subtask load_log_data
[2025-07-06T22:00:23.771+0000] {task_command.py:426} INFO - Running <TaskInstance: practice_etl_pipeline.load_log_data manual__2025-07-06T21:59:26.788588+00:00 [running]> on host 93b0937d74ff
[2025-07-06T22:00:23.822+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='practice_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_log_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-06T21:59:26.788588+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-06T21:59:26.788588+00:00'
[2025-07-06T22:00:23.823+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-06T22:00:28.913+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-06T22:00:28.914+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-06T22:00:28.919+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=practice_etl_pipeline, task_id=load_log_data, run_id=manual__2025-07-06T21:59:26.788588+00:00, execution_date=20250706T215926, start_date=20250706T220023, end_date=20250706T220028
[2025-07-06T22:00:28.947+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-06T22:00:28.956+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-06T22:00:28.959+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
